TowerForge/TowerForge/LevelManager/Components/HealthComponent.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  HealthComponent.swift
+//  TowerForge
+//
+//  Created by MacBook Pro on 15/03/24.
+//
+
+import Foundation
+
+class HealthComponent: TFComponent {
+    public var currentHealth: Int
+    public var maxHealth: Int
+    
+    init(maxHealth: Int) {
+        self.currentHealth = maxHealth
+        self.maxHealth = maxHealth
+        super.init()
+    }
+    
+    func decreaseHealth(amount: Int) {
+        self.currentHealth -= amount
+    }
+    
+    func increaseHealth(amount: Int) {
+        self.currentHealth = max(self.currentHealth + amount, self.maxHealth)
```

Comment:
Should this not be min? So that the health is capped by the max health?

Reply:
updated